source/assets/stylesheets/styles/_blog_article.scss

```diff
@@ -0,0 +1,279 @@
+    padding: 40px 0;
+  }
+}
+
+.blog-header__article-title {
+  font-size: 2.1em;
+  font-weight: bold;
+  line-height: 1.3em;
+  padding-bottom: 30px;
+
+  @include breakpoint($blog-container-width) {
+    padding-bottom: 50px;
+  }
+}
+
+.blog-header__article-author {
+  color: $author-name-color;
+  display: inline;
+  font-size: 1em;
+  letter-spacing: 0.02em;
+  padding-left: 2px;
+  text-decoration: none;
+}
+
+.blog-header__article-published-date {
+  color: $article-published-date;
+  display: inline;
+  font-size: 0.8em;
+
+  &:before {
+    content: "-";
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+}
+
+.blog-header__article-tags {
+  display: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: block;
+    padding-top: 40px;
+  }
+}
+
+.blog-header__article-tag-link {
+  @include breakpoint($tablet-breakpoint) {
+    display: inline-block;
+    padding-left: 15px;
+    padding-right: 15px;
+
+    a {
+      color: $medium-gray;
+      line-height: 3em;
+      text-decoration: none;
+    }
+  }
+}
+
+.blog-header__article-tag-active {
+  @include breakpoint($tablet-breakpoint) {
+    a {
+      color: $light-purple;
+    }
+  }
+}
+
+.blog-header__article-tag-border {
+  @include breakpoint($tablet-breakpoint) {
+    border-bottom: 2px solid;
+    border-color: darken($light-gray, 10%);
+    position: relative;
+  }
+}
+
+.blog-header__article-tag-border-active {
+  @include breakpoint($tablet-breakpoint) {
+    border-bottom: 3px solid;
+    border-color: $medium-purple;
+    position: relative;
+    top: 2px;
+  }
+}
+
+
+.blog-article {
+  background-color: $article-background-color;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.blog-article__container {
+  padding: 0 30px 10px;
+
+  @include breakpoint($tablet-breakpoint) {
+    padding-top: 15px;
+  }
+
+  @include breakpoint($blog-container-width) {
+    margin: 0 auto;
+    max-width: $blog-container-width;
+    padding-bottom: 50px;
+  }
+}
+
+.blog-article__content {
+  box-sizing: border-box;
+
+  p,
+  ul,
+  ol {
+    font-weight: 100;
+    line-height: 1.5;
+    padding-bottom: 26px;
+  }
+
+  a {
+    color: $article-link-text-color;
+    font-weight: 400;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  strong {
+    font-weight: 400;
+  }
+
+  em {
+    font-style: italic;
+  }
+
+  blockquote {
+    &:before {
+      background-color: $blockquote-white-background-color;
+      content: '';
+      display: block;
+      height: 30px;
+    }
+
+    background-color: $blockquote-background-color;
+    margin: -30px;
+
+    p {
+      padding: 30px 60px;
+    }
+
+    margin-bottom: 26px;
+  }
+
+  img {
+    box-sizing: border-box;
+    padding: 20px 0;
+    width: 100%;
+  }
+
+  li {
+    font-weight: 100;
+    line-height: 1.4;
+    margin-left: 3px;
+
+      &:before {
+        content: "•";
+        padding-right: 5px;
+      }
+  }
+
+  ol li {
+    list-style-type: decimal;
+  }
+
+  h2 {
```

Comment:
We don't have any styles for a `h1` tag? Though it might be worth investigating whether when we write:

```Markdown
# Heading 1
## Heading 2
```
it produces:
```HTML
<h2>Heading 1</h2>
<h3>Heading 2</h3>
```
as we should only have one `h1` tag which will be the blog post title.